pkg/breakglasscredential/flags.go

```diff
@@ -0,0 +1,144 @@
+package breakglasscredential
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	usernameFlag   = "username"
+	expirationFlag = "expiration"
+)
+
+type BreakGlassCredentialArgs struct {
+	username           string
+	expirationDuration time.Duration
+}
+
+func AddBreakGlassCredentialFlags(cmd *cobra.Command, prefix string) *BreakGlassCredentialArgs {
+	args := &BreakGlassCredentialArgs{}
+
+	cmd.Flags().StringVar(
+		&args.username,
+		usernameFlag,
+		"",
+		"Username for the break glass credential.",
+	)
+
+	cmd.Flags().DurationVar(
+		&args.expirationDuration,
+		expirationFlag,
+		0,
+		fmt.Sprintf("Expire the break glass credential after a relative duration like 2h, 8h. "+
+			"The expiration duration needs to be at least 10 minutes from now and to be at maximum 24 hours."),
+	)
+	return args
+}
+
+func GetBreakGlassCredentialOptions(cmd *pflag.FlagSet, prefix string,
```

Comment:
@chenz4027 `prefix` and `confirmBeforeAllArgs` parameters are not being used, so shall we remove them?

Reply:
@robpblake yep I will remove them